rotkehlchen/chain/ethereum/modules/open_ocean/decoder.py

```diff
@@ -0,0 +1,5 @@
+from rotkehlchen.chain.evm.decoding.open_ocean.decoder import OpenOceanCommonDecoder
+
+
+class OpenOceanDecoder(OpenOceanCommonDecoder):
+    pass
```

Comment:
Can you do

```suggestion
class OpenOceanDecoder(OpenOceanCommonDecoder):
    ...
```

instead of pass here and in the others?